returnn/config.py

```diff
@@ -149,6 +205,14 @@ def serialize(self):
         python_prolog_code = self.__parse_python(self.python_prolog)
         python_epilog_code = self.__parse_python(self.python_epilog)
 
+        if self.staged_network_dict:
+            config_lines.append(self.GET_NETWORK_CODE)
```

Comment:
If we add it to prolog before serializing, we could also define it as a function instead of a string which is somewhat nicer. However, then the function content is not hashed by default, which is why this might nevertheless not be ideal.